AUTHORS

```diff
@@ -0,0 +1,2 @@
+baghbidi <Kalpase_87>
```

Comment:
Could you please put my email address here: `saeid@edgeworx.io`
Don't know how this file is generated. It might be better to have our full name here.